tests/testthat/test-api-endpoints.R

```diff
@@ -240,6 +240,7 @@ test_that("graph cases averted config", {
 
 
 test_that("impact docs", {
+  knitr::pandoc(mintr_path("json/impact_docs.md"))
```

Comment:
As the text does not contain any R markdown, it'll be easier to automate possibly with commonmark:

```
txt <- readLines(mintr_path("json/impact_docs.md"))
commonmark::markdown_html(txt)
```

This also needs adding to the app startup so that the class there holds the processed strings ready to go, I think.

With that, all the pandoc hassle goes away too

Reply:
I feel like the `knitr` interface is more convenient - `commonmark` returns text, so would have to write this back to a file, rather than it being a one liner here. Is there some other reason to prefer `commonmark`? Also not too sure what you mean about app startup - seems preferable to compile as part of the [docker build](https://github.com/mrc-ide/mintr/pull/28/commits/351b2abd766b82dcb304aa3e5360b9b79452f386#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddcR28)) so that we don't add any time to the container startup (although these are small basic files so it's actually very quick, so let me know if there's some other reason for doing it that way I've overlooked)

Reply:
there's no need to roundtrip via a file since we want to return a string anyway. We already do a little work on app startup, so with the commonmark approach we'd process the md files into md strings, then the endpoints just return the strings.

It's also bad practice (and does not always work) to change files in the package tree (which the tests do here). I have a failing package on CRAN at the moment that I have to update because of this :(

With knitr way we also add quite a bit of size to the docker build with pandoc+knitr (not a dealbreaker but it's a bunch of stuff).

Reply:
If you'd rather not make the changes, we can merge this and I can work up a PR showing what I mean

Reply:
> no need to roundtrip via a file since we want to return a string anyway

Ah I see what you mean. So maybe doing that in the [`main` function](https://github.com/mrc-ide/mintr/blob/master/R/main.R#L24) where we instantiate the db?

Reply:
yes, possibly by putting them here: https://github.com/mrc-ide/mintr/blob/master/R/db.R#L20-L24 as part of the initialisation that happens there?